tests/test_objects.py

```diff
@@ -223,6 +226,19 @@ def test_texture():
     assert texture is not None
 
 
+def test_texture_repr():
+    texture = pyvista.Texture(examples.mapfile)
+    tex_repr = str(repr(texture))
```

Comment:
`str` should be a no-op here I think:
```suggestion
    tex_repr = repr(texture)
```